testing/ef-tests/src/models.rs

```diff
@@ -313,11 +313,11 @@ pub enum ForkSpec {
     Unknown,
 }
 
-impl From<ForkSpec> for ChainSpec {
-    fn from(fork_spec: ForkSpec) -> Self {
+impl From<&ForkSpec> for ChainSpec {
```

Comment:
hmm, converting this & does make much sense for copy types, especially if we do `*fork_spec` later